jcl/src/java.base/share/classes/java/security/AccessControlContext.java

```diff
@@ -733,10 +760,6 @@ private boolean debugHelper(Permission perm) {
  *                  if perm is null
  */
 public void checkPermission(Permission perm) throws AccessControlException {
-/*[IF JAVA_SPEC_VERSION >= 24]*/
-	/*[MSG "K002e", "checking permissions is not supported"]*/
-	throw new AccessControlException(Msg.getString("K002e")); //$NON-NLS-1$
-/*[ELSE] JAVA_SPEC_VERSION >= 24 */
```

Comment:
Please not remove this and other recent merged changes.

Reply:
updated